### PR TITLE
feat(data-explorer): display data types

### PR DESCRIPTION
**Pull request by MaxPostema**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
-  User documentation updated
- [x] Conventional commits (squash if needed)
- [x] No warnings during install
- Updated javascript typing

### CODE REVIEW

**Review by jvanveen — APPROVED** [review]
Looks good!

File: packages/data-explorer/src/components/dataView/TableRow.vue [hunk 1]
```diff
@@ -28,11 +28,64 @@
         <font-awesome-icon icon="trash"></font-awesome-icon>
       </button>
     </th>
-    <td v-for="(column, index) in visibleColumns" :key="index" class="text-nowrap text-truncate mg-data-column">
-      {{rowData[column.name]}}
+    <td v-for="(column, index) in visibleColumns" :key="index" class="mg-data-column">
+      <DataDisplayCell v-if="typeof getColumnName(column.name) != 'undefined'" :value="getColumnName(column.name)" :type="column.type" :data="column" :rowIndex="rowIndex"></DataDisplayCell>
```
Comment (connoratrug) [inline packages/data-explorer/src/components/dataView/TableRow.vue:32]: when does a column not have a name , and why would the cell be hidden en that case ?

Comment (MaxPostema) [reply packages/data-explorer/src/components/dataView/TableRow.vue:32]: column.name is the displayed value (like a date or a string). It's not possible to display a value that is undefined.

Comment (connoratrug) [reply packages/data-explorer/src/components/dataView/TableRow.vue:32]: 'column.name is the displayed value'  ? strange

**Pull request merged**